test/protocol_test.py

```diff
@@ -3446,8 +3448,8 @@ def test_transfer_informatics(self):
         assert new_instructions[3].informatics[0].compounds[0] == compd2
         wells = []
         for instr in new_instructions:
-            wells.append(instr.informatics[0].wells)
-        assert WellGroup(wells) == test_wells
+            wells.extend(instr.informatics[0].wells)
```

Comment:
Ah ok, so this was somehow working before because we were calling `append` which was appending a list to the list, which is wrong but will compile. 

`append` can take in a list or a single element. `extend` cannot though, so this test will break if `wells` is not a list. Should we modify this test to have a combination of instructions with multiple wells and instructions with single well?

Reply:
Sounds good. Updated test :)